test/touch_test.py

```diff
@@ -40,13 +41,24 @@ def test_num_fingers__invalid(self):
 class TouchInteractiveTest(unittest.TestCase):
     __tags__ = ["interactive"]
 
-    @unittest.skipIf(not has_touchdevice, "no touch devices found")
+    @classmethod
+    def setUpClass(cls):
+        # if we don't have a touch device, enable the mouse to act as a
+        # touch device for the interactive tests.
+        os.environ["SDL_MOUSE_TOUCH_EVENTS"] = "0"
+        pygame.init()
+        if touch.get_num_devices() == 0:
+            pygame.quit()
+            os.environ["SDL_MOUSE_TOUCH_EVENTS"] = "1"
+            pygame.init()
+
+    @classmethod
+    def tearDownClass(cls):
+        pygame.quit()
+
     def test_get_finger(self):
         """ask for touch input and check the dict"""
 
-        pygame.display.init()
-        pygame.font.init()
-
         os.environ["SDL_VIDEO_WINDOW_POS"] = "50,50"
         screen = pygame.display.set_mode((800, 600))
```

Comment:
```suggestion
        screen = pygame.display.set_mode((1000, 600))
```
minor update, 800 is too small for the event text being displayed